source/material_model/steinberger.cc

```diff
@@ -769,6 +769,15 @@ namespace aspect
           out.entropy_derivative_temperature[i] = 0;
           for (unsigned int c=0; c<in.composition[i].size(); ++c)
             out.reaction_terms[i][c]            = 0;
+
+          // fill seismic velocities outputs if they exist
+          SeismicAdditionalOutputs<dim> *seismic_out = out.template get_additional_output<SeismicAdditionalOutputs<dim> >();
+
+          if (seismic_out != NULL)
```

Comment:
there's a neat idiom that one can use here:
```
if (SeismicAdditionalOutputs<dim> *seismic_out = out.template get_additional_output<SeismicAdditionalOutputs<dim> >())
```
This avoids leaking the variable outside the scope that it's actually used in.

Makes no difference here, of course because the function doesn't extend past the if-block, but I believe in setting good examples :-)